solutions/inprompt.py

```diff
@@ -0,0 +1,84 @@
+import logging
```

Comment:
For this I've implemented less ideal soultion, shorter. Will add this as option 2, and yours will be main one. Thanks for contribution :) :)

```python
import logging
from lib.variables import get_environment_variable
from lib.openaiutils import openai_get_authenticated_client, get_openai_models


logger = logging.getLogger(__name__)

def _construct_chat_question(person_question, person_description):
    json_data = [
        {
            "role": "system",
            "content":f'''
In Polish language you will be asked about one person
Using one and only one word please answer about this person.

The person description is: {person_description}
'''
        },
        {
            "role": "user",
            "content": f'''Question is {person_question}'''
        }
    ]
    logger.info(f"TEXT: {json_data}")
    return json_data



def get_answer(task_details_response: dict):
    client = openai_get_authenticated_client()
    logger.info("Solving lesson 5 input task")
    question = task_details_response['question']
    message = task_details_response['msg']
    input_text = task_details_response['input']
    name = question.split(" ")[-1].replace("?","")
    table_new = []

    for t_name in input_text:
        if name in t_name:
            table_new.append(t_name)
    models = get_openai_models()
 
    task_response = client.chat.completions.create(
        model=models.gpt_3_5_turbo.name,
        messages=_construct_chat_question(question, table_new[0]),
        temperature=1,
        max_tokens=64,
        top_p=1
    )

    logger.debug(f"OpenAI Response { task_response }")

    return task_response.choices[0].message.content

```